core/src/datayoga_core/blocks/relational/write/block.py

```diff
@@ -7,17 +8,38 @@
 from datayoga_core.context import Context
 from datayoga_core.result import Result
 from sqlalchemy import Table
-from sqlalchemy.dialects.postgresql import insert
+from sqlalchemy.sql.expression import ColumnCollection
 
 logger = logging.getLogger("dy")
 
 
+@unique
+class DbType(Enum):
+    MYSQL = "mysql"
+    PSQL = "postgresql"
+
+
+def get_driver_name(db_type: str) -> str:
+    if db_type == DbType.MYSQL.value:
```

Comment:
can we turn this into a dict and just pull from there? we will have tens of them